pkg/reporter/reporter.go

```diff
@@ -204,7 +204,11 @@ func buildPrettyViolationsTable(violations []report.Violation) string {
 		table.Append([]string{yellow("Location:"), cyan(violation.Location.String())})
 
 		if violation.Location.Text != nil {
-			table.Append([]string{yellow("Text:"), strings.TrimSpace(*violation.Location.Text)})
+			if len(*violation.Location.Text) > 77 {
```

Comment:
I suggested 120 characters in the issue. Was there some specific reason you went with 80? Totally fine with me, just curious 🙂

Reply:
Ahh, I can change it. I got distracted by trying to use `tput cols` and then opted for a static value. I'll make it 120.